releases/v0.1.0/BMI-App_2025_clean-0.1.0/app.py

```
@@ -10,8 +10,7 @@
 
 try:
     pass  # type: ignore
-
-except Exception:
+except Exception:  # nosec B110
     # dotenv is optional; skip if missing in test env
     pass
```

Ruff (0.13.3)

`try`-`except`-`pass` detected, consider logging the exception

(S110)